extensions/azure/cosmos/cosmos-common/src/test/java/org/eclipse/dataspaceconnector/cosmos/azure/CosmosDbApiImplIntegrationTest.java

```diff
@@ -132,4 +134,25 @@ void queryItems() {
         assertThat(result).hasSize(2);
     }
 
+    @Test
+    void deleteItem_whenItemPresent_deleted() {
+        var testItem = new TestCosmosDocument("payload", PARTITION_KEY);
+        container.createItem(testItem);
+        var deletedItem = cosmosDbApi.deleteItem(testItem.getId());
+
+        assertThat(deletedItem).isNotNull().isInstanceOf(LinkedHashMap.class);
```

Comment:
why test for a specific type here? Should we test for Map?

Reply:
also isNotNull is redundant with isInstanceOf

Reply:
Ah yeah makes sense, I changed it to Map.